src/projects/EnsureThat/CollectionArg.cs

```diff
@@ -382,5 +382,50 @@ public T[] HasAny<T>([ValidatedNotNull]T[] value, [NotNull] Func<T, bool> predic
 
             return value;
         }
+
+        [NotNull]
+        [DebuggerStepThrough]
+        public IList<T> Contains<T>([NotNull, ValidatedNotNull]IList<T> value, T item, [InvokerParameterName] string paramName = Param.DefaultName)
```

Comment:
[NotNull, ValidatedNotNull] -> [ValidatedNotNull] as done in previous change

Reply:
Sigh, maybe I misunderstood this. It looks like it'll warn about passing in a null to this, but it doesn't seem to properly understand ValidatedNotNull. This will require more investigation for the proper combination, but shouldn't be a major issue for any consumers.